### ISSUE TITLE
Files without explicit licensing

### ISSUE THREAD

**Issue by xambroz** [radare/valabind#33]
Hello,
the package review of the valabind for Fedora revealed that there are files where it is not easy to review the licensing terms.

Please could you consider adding the text suggested in the LICENSE by the FSF to the beginning of each source file to make the licensing review easier?

Fedora valabind package review: https://bugzilla.redhat.com/show_bug.cgi?id=1270630

Template recommended by FSF:
```
    Copyright (C) <year>  <name of author>

    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Files with unclear licensing:
```
Unknown or generated
--------------------
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/foo.vapi
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/t/java/AsmExample.java
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/t/java/RAsm.java
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/t/java/RAsmCode.java
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/t/python/example.py
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/t/r2/test.cs
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/t/r2/test.java
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/t/r2/test.pl
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/t/r2/test.py
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/t/r2/test.rb
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/t/vala/test.cxx
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/t/vala/test.py
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/t/vala/vstest.vala

Copyleft 2009-2012 -- pancake // nopcode.org
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/cxxwriter.vala
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/girwriter.vala
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/valabindwriter.vala

Copyleft 2014 -- williballenthin
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/gowriter.vala

Copyleft 2012-2014 -- pancake, eddyb, ritesh
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/nodeffiwriter.vala

Copyleft 2009-2015 -- pancake, ritesh
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/utils.vala

Copyleft 2015 -- pancake
valabind-cd4051f6a7f63b297f3950c11a30f468351cbd69/windows.c
```

Thank you 
Michal Ambroz

**Comment by radare** [comment]
Thanks for reporting

All code in here is licensed under GPLv3, if files doesnt contain any license in header, the license is inherited from the project root directory

**Comment by xambroz** [comment]
Thank you.

**Comment by radare** [comment]
As far as i know it is ok to have a oneliner version message noting the authorship and alternative license. I'm changing all those copyleft ones for copyright, as long as its under GPLv3. This program is suposed to be used as an external tool, so i dont think that license is gonna be problematic for any use.

**Comment by radare** [comment]
Just pushed a commit now with that license fix (bumping to 0.9.3)

**Comment by xambroz** [comment]
Thank you.
